app/src/main/java/com/napzak/market/presentation/marketinfo/component/MarketTradeTypeTab.kt

```diff
@@ -0,0 +1,127 @@
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.napzak.market.core.common.extension.noRippleClickable
+import com.napzak.market.core.designsystem.theme.NapzakMarketTheme
+import com.napzak.market.core.type.TradeType
+
+/**
+ * 상품 거래 타입을 선택하는 탭 컴포넌트
+ *
+ * @param tradeTypeList 탭에 넣을 타입 리스트
+ * @param itemPadding 탭 사이에 적용될 패딩
+ * @param selectedTab 선택된 탭
+ * @param onTradeTypeClick 탭이 클릭 되었을 경우 실행되는 콜백
+ * @param modifier 수정자
+ */
+
+@Composable
+fun MarketTradeTypeTab(
+    tradeTypeList: List<TradeType>,
+    itemPadding: Dp,
+    selectedTab: TradeType,
+    onTradeTypeClick: (TradeType) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val borderColor = NapzakMarketTheme.colors.gray300
+
+    Row(
+        modifier = modifier
+            .drawBehind {
+                val strokeWidth = 2.dp.toPx()
+                val strokeHeightPoint = size.height - 1
+                drawLine(
+                    color = borderColor,
+                    start = Offset(0f, strokeHeightPoint),
+                    end = Offset(size.width, strokeHeightPoint),
+                    strokeWidth = strokeWidth,
+                )
+            }
+            .padding(horizontal = 20.dp),
+    ) {
+        tradeTypeList.forEachIndexed { index, tradeTypeItem ->
+            TradeTypeTabItem(
+                tradeType = tradeTypeItem.label,
+                isSelected = selectedTab == tradeTypeItem,
+                onTradeTypeClick = { onTradeTypeClick(tradeTypeItem) },
+                modifier = Modifier.weight(1f),
+            )
+
+            if (index != tradeTypeList.lastIndex) {
+                Spacer(Modifier.width(itemPadding))
+            }
+        }
+    }
+}
+
+@Composable
+private fun TradeTypeTabItem(
+    tradeType: String,
+    isSelected: Boolean,
+    onTradeTypeClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val borderColor = if (isSelected) NapzakMarketTheme.colors.purple30 else NapzakMarketTheme.colors.gray300
+    val textColor = if (isSelected) NapzakMarketTheme.colors.purple30 else NapzakMarketTheme.colors.gray600
+
+    Column(
+        modifier = modifier
+            .drawBehind {
+                val strokeWidth = 2.dp.toPx()
+                val strokeHeightPoint = size.height - 1
+                drawLine(
+                    color = borderColor,
+                    start = Offset(0f, strokeHeightPoint),
+                    end = Offset(size.width, strokeHeightPoint),
+                    strokeWidth = strokeWidth,
+                )
+            }
+            .noRippleClickable { onTradeTypeClick() }
```

Comment:
p2: 이렇게 코드 수정이 가능해요!
```suggestion
            .noRippleClickable(onTradeTypeClick)
```